tests/unit/customizations/eks/test_update_kubeconfig.py

```diff
@@ -217,7 +217,7 @@ def setUp(self):
         self._session.create_client.return_value = self._mock_client
         self._session.profile = None
 
-        self._client = EKSClient(self._session, parsed_args=Namespace(cluster_name="ExampleCluster", role_arn=None))
+        self._client = EKSClient(self._session, parsed_args=Namespace(cluster_name="ExampleCluster", role_arn=None, exclude_profile=False))
```

Comment:
```suggestion
        self._client = EKSClient(self._session, parsed_args=Namespace(cluster_name="ExampleCluster"))
```
Just realized that I set a bad example here. Since these are just default values, they don't need to be specified here.

Reply:
I thought so too, but the test actually fails without this - I'm not sure why defaults aren't being passed here; the functional tests & running the cli via an egg both work properly

Reply:
Trying your suggestion I get ```AttributeError: 'Namespace' object has no attribute 'role_arn'``` (I would get the same thing respectively with `exclude_profile`

Reply:
Ah understood, this is a Namespace from a clean slate, so it doesn't know about the args we defined